uwsgi/run.sh

```diff
@@ -21,7 +21,14 @@ elif [ $1 == "cleanup-db" ]; then
         echo "\${MAX_AGE} must be set!"
         exit 1
     fi
-    exec python scripts/manage-db.py -d ${DBURI} cleanup ${MAX_AGE}
+    if [ -z "${DELETE_RUN_TIME}" ]; then
+        echo "\${DELETE_RUN_TIME} must be set!"
+        exit 1
+    fi
+
+    exec scripts/run-batch-deletes.sh $DBURI $MAX_AGE $DELETE_RUN_TIME
+
+    #exec python scripts/manage-db.py -d ${DBURI} cleanup ${MAX_AGE}
 elif [ $1 == "cleanup-db-dryrun" ]; then
```

Comment:
It's just weird that "cleanup-db" and "cleanup-db-dryrun" do different things. If the latter exists, it should run through the same code paths (minus the DELETEs). I'm fine with just removing it though.